modular_skyrat/modules/advanced_shuttles/code/computer.dm

```diff
@@ -1,10 +1,18 @@
 /obj/machinery/computer/shuttle/pod/advanced
 	icon = 'modular_skyrat/modules/advanced_shuttles/icons/computer.dmi'
 	icon_state = "intercom"
+	icon_screen = "null"
 	layer = ABOVE_OBJ_LAYER
 
 	connectable = FALSE //connecting_computer change: since icon_state is not a typical console, it cannot be connectable.
 
+/obj/machinery/computer/shuttle/pod/advanced/emag_act(mob/user)
+	if(obj_flags & EMAGGED)
+		return
+	obj_flags |= EMAGGED
+	locked = FALSE
+	to_chat(user, span_warning("You fry the pod's alert level checking system."))
+
```

Comment:
Is this used any more? This is specifically the pod/advanced we had on our unique-sprited pods that we axed a while ago, and I dont see them used in any .dmm or other file.
If it IS unused, you can likely just cut it instead of redoing the emag_act and all this